net/respondd-module-lldp/Makefile

```diff
@@ -0,0 +1,37 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=respondd-module-lldp
+PKG_VERSION:=1
+PKG_RELEASE:=1
+
+PKG_BUILD_DIR := $(BUILD_DIR)/$(PKG_NAME)
+PKG_BUILD_DEPENDS := respondd lldpd
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/respondd-module-lldp
+  SECTION:=gluon
+  CATEGORY:=Gluon
+  TITLE:=adds lldp usage neighbours provider to respondd
+  DEPENDS:=+respondd +lldpd
+endef
+
+define Build/Prepare
+	mkdir -p $(PKG_BUILD_DIR)
+	$(CP) ./src/* $(PKG_BUILD_DIR)/
+endef
+
+define Build/Configure
+endef
```

Comment:
The default Build/Configure does nothing when no configure script exists, so this can be dropped.